pkgs/tools/wayland/hyprevents/default.nix

```
@@ -0,0 +1,40 @@
+{ fetchgit, lib, makeWrapper, stdenv, hyprland, bash }:
+
+stdenv.mkDerivation rec {
+  pname = "hyprevents";
+  version = "1.0-20221209";
+
+  src = fetchgit {
+    url = "https://github.com/vilari-mickopf/hyprevents";
+    rev = "95426ae4dac70c6e6ed67918dbf3bdedfdf28f37";
+    sha256 = "sha256-SLt2ukCJbfkvnGLgy9x7/3BoI2OqC4VIdgM5z768EXc=";
```

Prefer `hash` over `sha256`

```suggestion
    hash = "sha256-SLt2ukCJbfkvnGLgy9x7/3BoI2OqC4VIdgM5z768EXc=";
```